frontend/amundsen_application/static/js/ducks/tableMetadata/api/helpers.ts

```diff
@@ -44,14 +44,15 @@ export function getRelatedDashboardSlug(key: string): string {
  */
 export function processColumns(
   columns: TableColumn[],
+  tableKey: string,
   databaseId?: string
 ): TableColumn[] {
-  return columns.map((column, index) => {
+  return columns.map((column) => {
     const nestedType = parseNestedType(column.col_type, databaseId);
 
     return {
       ...column,
-      col_index: index,
+      key: tableKey + '/' + column.name,
```

Comment:
Do we have to rebuild the column key here? is it not indexed in neo4j?

Reply:
You're right that it is indexed, but it wasn't being returned from the metadata API. I thought about adding it to the neo4j proxy query but then realized that wouldn't guarantee that the key was available for those using other proxies, and this way would guarantee we'd always have the key in FE